packages/infra/lib/ihe-gateway-v2-stack.ts

```diff
@@ -67,6 +70,45 @@ export class IHEGatewayV2LambdasNestedStack extends NestedStack {
     documentRetrievalLambda.grantInvoke(props.apiTaskRole);
   }
 
+  private createParsedReponseTables(iheParsedResponsesBucket: s3.Bucket) {
+    new glue.CfnTable(this, "iheParsedResponsesDebugTable", {
+      catalogId: `arn:aws:glue:us-east-1:123456789012:catalog`,
+      databaseName: "default",
+      tableInput: {
+        description: "Table used for debugging IHE parsed responses",
+        name: "ihe_parsed_respones_debug_table",
+        partitionKeys: [
+          { name: "cxid", type: "string" },
+          { name: "patientid", type: "string" },
+          { name: "stage", type: "string" },
+          { name: "requestid", type: "string" },
+          { name: "gatewayoid", type: "string" },
+        ],
+        storageDescriptor: {
+          columns: [
+            { name: "id", type: "string" },
+            { name: "timestamp", type: "string" },
+            { name: "requesttimestamp", type: "string" },
+            { name: "responsetimestamp", type: "string" },
+            { name: "gateway", type: "struct<url:string,oid:string,id:string>" },
+            { name: "patientmatch", type: "string" },
+            { name: "ihegatewayv2", type: "boolean" },
+            {
+              name: "operationoutcome",
+              type: "struct<resourcetype:string,id:string,issue:array<struct<severity:string,code:string,details:struct<text:string>>>>",
+            },
+          ],
+          compressed: false,
+          inputFormat: "org.apache.hadoop.mapred.TextInputFormat",
+          outputFormat: "org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat",
+          location: `s3://${iheParsedResponsesBucket.name}/`,
```

Comment:
build error. this should be iheParsedResponsesBucket.bucketName

Reply:
had fixed it, but hadn't pushed. Fixed.